docs/HUs.md

```diff
@@ -1,7 +1,8 @@
 # User Stories
 
 ## HU001:
-Uno de los problemas de los estudiantes a lo largo de su estancia en la Escuela, es la desorganización, al dividir las asignaturas en teoría y prácticas, los alumnos deben tener en cuenta varios detalles a la hora de ir a clase para la cuál no hay una manera sencilla de resolverlos. Aunque la forma regular de paliar ésto, es buscar por interntet, al proceso se le añade la infinidad de webs, enlaces y documentos entre las que se reparten las posibles respuestas.
+Uno de los problemas de los estudiantes a lo largo de su estancia en la Escuela, es la desorganización, al dividir las asignaturas en teoría y prácticas, los alumnos deben tener en cuenta varios detalles a la hora de ir a clase para la cuál no hay una manera sencilla de resolverlos.
+La forma actual de paliar el problema es buscar por interntet, al proceso se le añade la infinidad de webs, enlaces y documentos entre las que se reparten las posibles respuestas.
```

Comment:
Afú. No pongáis cambios que no se piden, por favor. No he dicho nada sobre esto, ¿no?

Reply:
No, no pero para que quede más claro (aunque en el preview se ve practicamente igual).